integration.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
```

Comment:
formerly `run_integration.sh`. Some extra logic has been added to:
1. clone the sdk repo as configured by `$TESTING_BRANCH` + `$TESTING_URL`
2. only dockerize the python sdk when `$DOCKERIZE_PYSDK==true`
3. pass the `UP_ENV` file directly to `up.sh`
4. only run all the tests via docker when `$DOCKERIZE_PYSDK==true`